js/src/volume.js

```diff
@@ -246,6 +246,9 @@ var ScatterView = widgets.WidgetView.extend( {
                 animation_time_vz : { type: "f", value: 1. },
                 animation_time_size : { type: "f", value: 1. },
                 animation_time_color : { type: "f", value: 1. },
+                animation_time_index : { type: "f", value: 1. },
```

Comment:
I agree but I don't know how to trigger the animation.
If I remove this line the changes of sequence_index are not reflected.